new-findings.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
```

Comment:
Could you run `shellcheck` against this script and fix any issues?

Reply:
Done.